requirements.txt

```
@@ -1,6 +1,6 @@
-docutils==0.16
+docutils==0.17
```

Sphinx 6.1.3 is run and requires 0.18, but sphinx_rtd_theme only works up to docutils 0.17.
```suggestion
docutils==0.18
```

---

found out that 5.3.0 was the last one to support up to 0.17. It might even allow upto docutils 0.18, but looks like checks are passing now.